.github/workflows/unit-tests.yml

```diff
@@ -129,7 +129,7 @@ jobs:
 
       - name: Install PHP dependencies
         run: |
-          if [[ "${{ matrix.type }}" != Phpunit* ]; then composer remove --no-interaction --no-update phpunit/phpunit johnkary/phpunit-speedtrap phpunit/dbunit phpunit/phpcov --dev ; fi
+          if [[ "${{ matrix.type }}" != Phpunit* ]]; then composer remove --no-interaction --no-update phpunit/phpunit johnkary/phpunit-speedtrap phpunit/dbunit phpunit/phpcov --dev ; fi
```

Comment:
Is the typo not the `[[`?

Reply:
no, because of `*`, bacause of bash...

Reply:
failing because GH probably uses sh instead of bash, debugging it....